CharacterCreation/00_New_Player_Walkthrough.txt

```diff
@@ -0,0 +1,47 @@
+## Step 0: Ask for Advice
+No really, if you get stuck at any point or just want a second opinion, other players can be tremendously helpful and
+are usually more than happy to assist new players getting their feet wet.
+
+## Step 1: Race
+*Section to be added on account of impending large-scale racial rework*
+
+## Step 2: Class
+For players new to role-playing games we recommend the Soldier or Hitman classes (and possibly Pilot).
+*Section to be expanded later on account of impending large-scale class rework*
+
+## Step 3: Background and Skills
+*Section to be added on account of impending addition of this feature*
+
+## Step 4: Attribute Point Allocation
+Now that you know what your character is capable of in-combat (your class) and out-of-combat (your non-combat skills), 
+you have all the information you need to decide your character's attributes. You have a total number of attribute points
+to allocate based on your choice of race, and 7 attributes to which to allocate them. Basic Rules.txt describes what each 
+attribute does in a vacuum if you need a refresher. Here are a few pointers:
+1. Unless you know what you are doing, do not give your character less than 6 FORT. FORT directly determines your health
+points, and having enough of them contributes to not dying.
+2. Have at least 6 in either STR or DEX, and if possible up to 8. One of these stats will determine your movement speed,
+which is of high value.
+3. Have at least 6 Luck. Luck is a unique attribute and its value can be hard to gauge without experiencing its effects 
+firsthand.
```

Comment:
I would suggest having at least 4 luck. I wouldn't say that your first character would need to be 'lucky'.

Reply:
Fair enough. I can settle for just advising them to not be actively unlucky.

Reply:
super nit-pick, the space is missing between "attribute pointes_to allocoate"